<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgFileIndexer</class>
 <widget class="QDialog" name="dlgFileIndexer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labDir">
       <property name="text">
        <string>&amp;Racine :</string>
       </property>
       <property name="buddy">
        <cstring>edtDir</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtDir"/>
     </item>
     <item>
      <widget class="QPushButton" name="pbuDir">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labExt">
       <property name="text">
        <string>&amp;Extension :</string>
       </property>
       <property name="buddy">
        <cstring>cbxExt</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbxExt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>mp3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mp4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>avi</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbuSearch">
       <property name="text">
        <string>Chercher</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tblFiles">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="labFiles">
         <property name="text">
          <string>Fichiers :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdFiles"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>taille totale</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="lcdSize"/>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="rbuMega">
            <property name="text">
             <string>&amp;Mo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="rbugiga">
            <property name="text">
             <string>&amp;Go</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pbuSave">
         <property name="text">
          <string>&amp;Sauvegarder</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbuQuit">
         <property name="text">
          <string>&amp;Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbuSave</tabstop>
  <tabstop>pbuQuit</tabstop>
  <tabstop>edtDir</tabstop>
  <tabstop>pbuDir</tabstop>
  <tabstop>cbxExt</tabstop>
  <tabstop>pbuSearch</tabstop>
  <tabstop>rbuMega</tabstop>
  <tabstop>rbugiga</tabstop>
  <tabstop>tblFiles</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pbuSearch</sender>
   <signal>clicked()</signal>
   <receiver>tblFiles</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>446</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbuQuit</sender>
   <signal>clicked()</signal>
   <receiver>dlgFileIndexer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
